### DISCUSSION TITLE
react-scripts not liking direct css import

### DISCUSSION THREAD

**Discussion by 0xModene** [rainbow-me/rainbowkit#656] [General]
In the past I've struggled with the same issue as #372 but was able to resolve with the same direct import solution. 

Now, I'm attempting to update our `react-scripts` dependency and am receiving this. Is there any solution here that rainbow devs have noticed?

FWIW, removing the direct css import mentioned in #372 resolves this error and my app compiles, but then I don't have any of that css. 

```bash
Compiled with problems:X

ERROR in ./src/index.tsx 40:0-47

Module not found: Error: Package path ./dist/index.css is not exported from package /Users/0xModene/dev/index-app/node_modules/@rainbow-me/rainbowkit (see exports field in /Users/0xModene/dev/index-app/node_modules/@rainbow-me/rainbowkit/package.json)

```

**Comment by 0xModene** [comment]
cc @markdalgleish @nickbytes

**Comment by markdalgleish** [comment]
Are you able to share a repro of the latest issue you're seeing?

**Reply by 0xModene** [reply]
It's a private repo, but I've added you to it. https://github.com/IndexCoop/index-app
